enhancements/builds/entitlements.md

```diff
@@ -0,0 +1,387 @@
+>on what the order of precedence will be (with the take on that listed below).
+> 4. There has also been a question in the past needing to shard entitlements due to API rate limiting.  To date, it
+>has been stated that is not a concern for builds.  Periodic checkpoints here could be prudent.
+> 5. There is also the general question of how entitlements are propagated for our managed service offerings?  Reaching 
+>out to the appropriate teams there to understand current capabilities/processes and building from there is needed.
+
+## Summary
+
+So in V3.x, all OpenShift Builds essentially had the customer RHEL entitlements mounted into the build pod automatically
+and without further intervention after install "for free" because:
+
+ - the ansible based installer required/forced providing your subscription information in order to install OCP
+ - for reference, see [these docs](https://docs.openshift.com/container-platform/3.11/install/host_preparation.html#host-registration) 
+ and [these roles](https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/rhel_subscribe/tasks/main.yml)
+ - next, Red Hats's docker daemon installed in 3.x had specific logic that auto-mounted the host's subscriptions 
+ into the container
+
+In V4.x, with 
+
+- the removal of the docker daemon, meaning no docker socket and no access to the host's content
+- and the total replacement of the openshift ansible based installer, so no gathering of entitlements during install
+
+all those 3.x mechanisms are gone, and a series of steps from either the cluster admin and/or developer attempting
+to run OpenShift Builds leveraging entitled content must be performed after install to enable access to the entitled
+content during the build.
+
+The highlights of those steps from [the current 4.x doc](https://docs.openshift.com/container-platform/4.3/builds/running-entitled-builds.html):
+
+1) First, either
+ 
+- create an ImageStreamTag to the UBI image from `registry.redhat.io` in the `openshift` namespace, or 
+- reference the UBI image directly and include a pull secret to `registry.redhat.io` in your BuildConfig
+
+2) Next, based on your access to the credentials:
+
+- With direct access to the entitlement pem files, create a generic secret from the files and reference the secret in
+your BuildConfig's input/source secret list
+- With Subscription Manager, and S2I build strategy, create secret(s) for the Subscription Manager configuration and 
+certs and reference the secret(s) in your BuildConfig's input/source secret list
+- With Subscription Manager, and Docker build strategy, you have to modify your Dockerfile and manually copy and 
+initialize as needed the information from the base image
+- With Satellite, and S2I build strategy, create a secret to reference the repository config file you created for your
+satellite instance and reference the secret in you BuildConfig's input/source secret list
```

Comment:
Another possibility is to subscribe a node with satellite (often the jump-host) and take the `*.pem` files from the file-system to create a secret.

Reply:
OK I've attempted to capture this in a *NOTE* following the list here.  Will be part of next push.